providers/websockets/kucoin/README.md

```diff
@@ -0,0 +1,13 @@
+# Kucoin Provider
```

Comment:
```suggestion
# KuCoin Provider
```

KuCoin is always stylized like that. We should probably do the same just to be cute.